.github/dependabot.yml

```diff
@@ -9,178 +9,238 @@ updates:
+    directory: "/extension/observer/k8sobserver"
     schedule:
       interval: "weekly"
+
+  #################
+  #   EXPORTERS   #
+  #################
   - package-ecosystem: "gomod"
-    directory: "/receiver/kubeletstatsreceiver"
+    directory: "/exporter/alibabacloudlogserviceexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/receiver/prometheusexecreceiver"
+    directory: "/exporter/awsemfexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/receiver/receivercreator"
+    directory: "/exporter/awsxrayexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/receiver/redisreceiver"
+    directory: "/exporter/azuremonitorexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/receiver/sapmreceiver"
+    directory: "/exporter/carbonexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/receiver/signalfxreceiver"
+    directory: "/exporter/datadogexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/receiver/simpleprometheusreceiver"
+    directory: "/exporter/elasticexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/receiver/statsdreceiver"
+    directory: "/exporter/honeycombexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/receiver/splunkhecreceiver"
+    directory: "/exporter/jaegerthrifthttpexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/receiver/wavefrontreceiver"
+    directory: "/exporter/kinesisexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/internal/common"
+    directory: "/exporter/logzioexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/internal/awsxray"
+    directory: "/exporter/newrelicexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/internal/k8sconfig"
+    directory: "/exporter/sapmexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/internal/awsxray/testdata/sampleapp"
+    directory: "/exporter/sentryexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/internal/awsxray/testdata/sampleserver"
+    directory: "/exporter/signalfxexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/extension/jmxmetricsextension"
+    directory: "/exporter/splunkhecexporter"
     schedule:
       interval: "weekly"
   - package-ecosystem: "gomod"
-    directory: "/extension/observer"
+    directory: "/exporter/stackdriverexporter"
     schedule:
       interval: "weekly"
+
+  ##################
+  #   PROCESSORS   #
```

Comment:
It's missing the `/processor/groupbytraceprocessor` in this list.

Reply:
It is here, I added in the second commit last night

Reply:
Indeed. The difference that I see is that this PR isn't based on the current master and lacks the processor itself. 

```console
$ find . -name go.mod | sed "s~^\./~/~" | sed "s/\/go.mod$//" | sort > /tmp/gomods.txt
$ grep directory .github/dependabot.yml | awk '{print $2}' | sed 's/"//gi' | sort > /tmp/gomods-dependabot.txt
$ diff /tmp/gomods.txt /tmp/gomods-dependabot.txt 
1c1
< 
---
> /
28a29
> /processor/groupbytraceprocessor

$ ls -1 processor/
k8sprocessor
metricstransformprocessor
resourcedetectionprocessor
routingprocessor

```

Reply:
Nice :)